models/models.js

```diff
@@ -27,10 +27,18 @@ exports.printEndpoints = () => {
 exports.getSingleArticlesComments = (inputId) => {
     const id = [ inputId ]
     return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`, id)
+    
     .then(({ rows })  => {
+        return rows
+    })
+}
+
+exports.checkArticleExists = (inputId) => {
+    const id = [ inputId ]
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, id)
+    .then(({rows}) => {
         if (rows.length === 0) {
             return Promise.reject({ status: 404, msg: 'not found'})
         }
```

Comment:
Does PSQL not give us an error back if the article isn't found?

Reply:
Oh nice I'll check - thank you!